users/users-model.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../data/dbConfig');
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById,
+  findByDept
+};
+
+function find() {
+  return db('users');
+}
+
+function findById(id) {
+  return db('users')
+    .where({ id })
+    .first();
+}
+
+async function add(user) {
+  const [id] = await db('users').insert(user);
```

Comment:
So there is nothing wrong with what you have here for sqlite3 HOWEVER when you switch to a PostgreSQL destructing id from an array, `[id]` , will throw an error